denops/ai-review/store/config.ts

```diff
@@ -29,11 +40,32 @@ export const configSlice = createSlice({
   initialState: configInitialState,
   reducers: {
     config: (state, action: PayloadAction<ConfigState>) => {
-      // TODO: validate config
-      state.config = action.payload.config
+      const input = action.payload.config
+      validate(input)
+      state.config = input
     },
   },
 })
 
-export const modelSelector = (): string => store.getState().config.config.chat_gpt.model
-export const logDirSelector = (): string => store.getState().config.config.log_dir
+const validate = (config: Config) => {
+  if (config.chat_gpt.model === "") {
+    throw new Error("chatGPT model is empty")
+  }
+  const azureConfig = config.chat_gpt.azure
+  if (azureConfig.use) {
+    console.log(azureConfig)
```

Comment:
Please delete console.log for print debugging

Reply:
I apologize for my careless mistake.
I deleted console.log.